internal/cache/profiles.go

```diff
@@ -29,7 +29,7 @@ type ProfileCache interface {
 	Remove(id string) error
 	RemoveByName(name string) error
 	DeviceResource(profileName string, resourceName string) (contract.DeviceResource, bool)
-	CommandExists(profileName string, cmd string) (bool, error)
+	CommandExists(profileName string, cmd string, method string) (bool, error)
```

Comment:
This is considered a breaking API change?

Reply:
AFAIK this function is only used inside SDK and it's in `internal` directory so it can't be imported by other service.

Reply:
@lenny-intel could you please review this PR again?